packages/studio/frontend/src/components/SuperInput/scripts/genDocTree.ts

```diff
@@ -28,7 +28,7 @@ const ARGS: TJS.PartialArgs = {
 const ROOT_NAME = 'event'
 const ROOT_TYPE = 'IO.IncomingEvent'
 
-const files = glob.sync('../server/src/sdk/*.ts')
+const files = glob.sync('../studio-be/src/sdk/*.ts')
```

Comment:
This path does not exist anymore since the SDK is now a sub-package of the runtime. We will have to rethink how we want to expose those typings properly since we want to also be able to load them in the code editor.

Reply:
we'll figure something out, the idea is to decommission this script, it was a temporary solution to a larger problem that has to be solved before we release Conv. Studio anyway.

Reply:
Perfect, I created a task for it: https://linear.app/botpress/issue/DEV-2638/[bug]-find-a-way-to-expose-the-sdk-for-the-code-editor-and-the-super